src/app/formly/radio/radio.module.ts

```diff
@@ -0,0 +1,58 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { DocumentationExamplesPage } from '../../shared/examples-page/examples.component';
+import {
+    DocumentationComponentsSharedModule,
+    DocumentationDemoList,
+} from '../../shared/index';
+import { DemoWrapperComponent } from '../../shared/demo-wrapper.component';
+import { FormlyBasicRadioComponent } from './demos/basic/radio-basic.component';
+import { FormlyBasicRadioModule } from './demos/basic/radio-basic.module';
+
+declare var require: any;
+
+const DEMOS = {
+    basic: {
+        title: 'Basic Radio',
+        type: FormlyBasicRadioComponent,
+        code: require('!!raw-loader!./demos/basic/radio-basic.component'),
+        markup: require('!!raw-loader!./demos/basic/radio-basic.component.html'),
+        module: require('!!raw-loader!./demos/basic/radio-basic.module'),
+        path: 'src/app/formly/radio/demos/basic',
+    },
```

Comment:
Could you add a footer section for these formly demos documenting available template options? I.E - a user won't know that they can pass in `checked` option to pre-check a radio or `tile` to display as tile without digging through the internal code

Reply:
Sure